internal/pkg/flags/flags.go

```diff
@@ -306,7 +319,14 @@ func (f FlagVar) Validate() error {
 		4096: true,
 		8192: false, // see https://github.com/kyma-project/lifecycle-manager/issues/1793
 	}[f.SelfSignedCertKeySize] {
-		return errInvalidSelfSignedCertKeyLenght
+		return errInvalidSelfSignedCertKeyLength
+	}
+
+	if f.ManifestRequeueJitterProbability <= 0 || f.ManifestRequeueJitterProbability >= 0.05 {
+		return errInvalidManifestRequeueJitterPercentage
+	}
+	if f.ManifestRequeueJitterProbability <= 0 || f.ManifestRequeueJitterProbability >= 1 {
```

Comment:
Actually I think both zero and one are totally good values here.

Reply:
Good catch. I somehow inverted the logic for the check in my head and was trying to include 0 and 1. I agree. They are valid opitons